src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -948,7 +948,7 @@ impl GraphEditorModelWithNetwork {
         frp::new_bridge_network! { [self.network, node.main_area.events.network]
             eval_ node.drag_area.events.mouse_down(touch.nodes.down.emit(node_id));
             eval  node.ports.frp.cursor_style ((style) cursor_style.emit(style));
-            eval_ node.frp.output_ports.mouse_down (output_press.emit(node_id));
+            eval_ node.view.output_ports.frp.port_mouse_down (output_press.emit(node_id));
```

Comment:
This is unsafe now. What happens if someone adds output port to a node? If I understand correctly, we are then regenerating the whole output port structure, right? So this binding will stop working. Isntead, we should expose frp endpoint in node frp for that and in case we change output t ports, we should rebind frp there.

Reply:
Hm. You are right if we regenerate the whole output area, then this would not work. But this endpoint could still work if we implement the changing of the ports internally to the `OutputPort` struct. Might be a question of how we want to handle the FRP design for the node in general: do we want to reroute, or give access to internal structs?

Reply:
I like the current design if we keep `output_ports.frp` as something which will automatically rebind internally when needed.